examples/Crameri2012_case2/generate_input_files.py

```diff
@@ -34,65 +37,70 @@
 # Interfaces (bottom first)
 ##############################################################################
 
-x_plume_left = int(((Lx - D_plume)/2) / (Lx/(Nx-1)))
-x_plume_right = int(((Lx + D_plume)/2) / (Lx/(Nx-1)))
+x_plume_left = int(((Lx - D_plume) / 2) / (Lx / (Nx - 1)))
+x_plume_right = int(((Lx + D_plume) / 2) / (Lx / (Nx - 1)))
 
 plume_bottom = -1 * np.ones(Nx) * (Lz - H_plume)
 plume_top = -1 * np.ones(Nx) * (Lz - H_plume)
 
-plume_bottom[x_plume_left:x_plume_right] = -1 *(Lz - H_plume) - np.sqrt(np.power(D_plume/2, 2) - np.power(x[x_plume_left:x_plume_right] - Lx/2, 2))
-plume_top[x_plume_left:x_plume_right] = -1 * (Lz - H_plume) + np.sqrt(np.power(D_plume/2, 2) - np.power(x[x_plume_left:x_plume_right] - Lx/2, 2))
+plume_bottom[x_plume_left:x_plume_right] = -1 * (Lz - H_plume) - np.sqrt(
+    np.power(D_plume / 2, 2) - np.power(x[x_plume_left:x_plume_right] - Lx / 2, 2)
+)
+plume_top[x_plume_left:x_plume_right] = -1 * (Lz - H_plume) + np.sqrt(
+    np.power(D_plume / 2, 2) - np.power(x[x_plume_left:x_plume_right] - Lx / 2, 2)
+)
 
 interfaces = {
-    'plume_bottom': plume_bottom,
-    'plume_top': plume_top,
-    'mantle_top': -1 * (np.ones(Nx) * (H_sa + H_lid)),
-    'lid_top': -1 * (np.ones(Nx) * (H_sa)),
+    "plume_bottom": plume_bottom,
+    "plume_top": plume_top,
+    "mantle_top": -1 * (np.ones(Nx) * (H_sa + H_lid)),
+    "lid_top": -1 * (np.ones(Nx) * (H_sa)),
 }
 
-with open('interfaces.txt', 'w') as f:
-    layer_properties = f'''
+with open("interfaces.txt", "w") as f:
+    layer_properties = f"""
         C   1.0    0.1    1.0    100.0  0.01
         rho 3300.0 3200.0 3300.0 3300.0 0.0
         H   0.0    0.0    0.0    0.0    0.0
         A   0.0    0.0    0.0    0.0    0.0
         n   0.0    0.0    0.0    0.0    0.0
         Q   0.0    0.0    0.0    0.0    0.0
         V   0.0    0.0    0.0    0.0    0.0
-    '''
+    """
 
-    for line in layer_properties.split('\n'):
+    for line in layer_properties.split("\n"):
         line = line.strip()
         if len(line):
-            f.write(' '.join(line.split()) + '\n')
+            f.write(" ".join(line.split()) + "\n")
```

Comment:
for line in layer_properties.split(" "):
        if len(line):
            f.write(" ".join(line.split()) + "\n")